boot/freeldr/freeldr/arch/i386/multiboot.S

```diff
@@ -139,10 +181,14 @@ gdt:
     .word HEX(9300)
     .word HEX(0000)
 
-    /* GDT pointer */
+/* GDT table pointer */
 gdtptr:
     .word HEX(17)       /* Limit */
+#ifndef _USE_ML
     .long gdt - 8       /* Base Address */
+#else
+    .long offset gdt - 8
```

Comment:
Doesn't the latter work for both?

Reply:
I just retested; `.long gdt - 8` works in both ML and GAS.
On the contrary, `.long offset gdt - 8` works in ML but breaks GAS with the error:
```
{standard input}: Assembler messages:
{standard input}: Fatal error: Case value 61 unexpected at line 1152 of file "../../../../build/binutils/src/gas/symbols.c"
```